main.go

```diff
@@ -74,18 +95,25 @@ func parseFlags(program string, args []string) (c *config, output string, err er
 
 	var conf config
 	flags.StringVar(&conf.gf, "gf", "", "Path to local geofeed file to verify")
+	flags.StringVar(&conf.isp, "isp", "", "Path to ISP MMDB file (optional)")
 	flags.StringVar(
 		&conf.db,
 		"db",
 		"/usr/local/share/GeoIP/GeoIP2-City.mmdb",
 		"Path to MMDB file to compare geofeed file against",
 	)
+	flags.BoolVar(&conf.version, "V", false, "Display version")
 
 	err = flags.Parse(args)
 	if err != nil {
 		return nil, buf.String(), err
 	}
 
+	if conf.version {
+		log.Printf("mm-geofeed-verifier %s", version)
```

Comment:
maybe this could use os.Executable() (or os.Args[0]) instead of the hard-coded 'mm-geofeed-verifier' name?

Reply:
I think I don't want to use those here; they print the name of the executable, I believe with the full path in both cases, which I'm not really sure I want to do in the case of a symlink or `go run`. In this particular case, it probably doesn't matter much either way, but I think in general I'd want to print out the 'name of the program' and the 'path to the executable'.

Reply:
filepath.Base(os.Args[0])  
should print just the name of the executable - not sure if it's worth the call, though:)